### PR TITLE
Add SimplifyBooleanExpression Checkstyle check

### PR DESCRIPTION
**Pull request by AustinShalit**

### CODE REVIEW

File: src/main/java/edu/wpi/first/shuffleboard/widget/BooleanBox.java [hunk 1]
```diff
@@ -43,7 +43,7 @@ private Color getColor() {
     final Boolean data = getData();
     if (data == null) {
       return Color.BLACK;
-    } else if (data == true) { //NOPMD
```
Comment (nightpool) [inline src/main/java/edu/wpi/first/shuffleboard/widget/BooleanBox.java]: this was added specifically for readability. please revert it and disable the checkstyle check for this line.

Comment (AustinShalit) [inline src/main/java/edu/wpi/first/shuffleboard/widget/BooleanBox.java]: I just pushed a commit that checks the equality using the equals method.

Comment (nightpool) [inline src/main/java/edu/wpi/first/shuffleboard/widget/BooleanBox.java]: But it's still not consistent with the line above it?

Comment (AustinShalit) [inline src/main/java/edu/wpi/first/shuffleboard/widget/BooleanBox.java]: Updated again.  Take a look

**Comment by nightpool** [comment]
Looks great!

**Pull request merged**